src/components/menu/component.tsx

```
@@ -0,0 +1,90 @@
+        if (e.target === e.currentTarget) {
+            onClose();
+        }
+    };
+
+    const handleResetSettings = () => {
+        browser.runtime
+            .sendMessage({
+                type: "RESET_SETTINGS",
+            })
+            .then((response) => {
+                if (response && response.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)